src/page-multi-facility/FacilityRow.tsx

```diff
@@ -85,6 +89,12 @@ interface Props {
   onSave: (f: Facility) => void;
 }
 
+function usedReferenceData(facility: Facility) {
+  const modelVersions = facility.modelVersions;
+  const isReferenceModelVersions = modelVersions.filter((v) => v.isReference);
+  return isReferenceModelVersions.length > 0;
```

Comment:
could also just get the boolean from `modelVersions.some((v) => v.isReference)` and not need to build a separate array here